README.md

```diff
@@ -35,6 +35,20 @@ Browsershot also can get the body of an html page after JavaScript has been exec
 Browsershot::url('https://example.com')->bodyHtml(); // returns the html of the body
 ```
 
+If you wish to retrieve a JSON list of all of the requests that the page triggered you can do so:
+
+```php
+$response = Browsershot::url('https://example.com')->triggeredRequest();
```

Comment:
Could you rename the method to `triggeredRequests()` and let it return an array?

Reply:
There you have it.